src/Controllers/news.controller.ts

```diff
@@ -0,0 +1,99 @@
+import {
+    Express,
+    Request,
+    Response,
+    NextFunction,
+    RequestHandler,
+} from 'express';
+import axios, { AxiosError } from 'axios';
+import cheerio from 'cheerio';
+import { News } from '@/Types/News';
+
+export const getNewsMFI: RequestHandler = async (
+    req: Request,
+    res: Response,
+    next: NextFunction
+) => {
+    try {
+        const HTMLDataRequest = await axios.get(
+            'https://mfi.ug.edu.pl/wydzial/aktualnosci'
+        );
+        const HTMLData = HTMLDataRequest.data;
+
+        const newsArray: News[] = [];
+        const $ = cheerio.load(HTMLData);
+        const selectedElem = '.item-list > ul > li';
+        $(selectedElem).map((parentIndex, parentElem) => {
+            const img = $(parentElem)
+                .find('.image-style-news-main-page')
+                .attr('src');
+            const href = $(parentElem).find('.news-title a').attr('href');
+            const title = $(parentElem).find('.news-title a').text();
+            const datetime = $(parentElem).find('.news-datetime').text();
+            const body = $(parentElem).find('.news-body').text();
+            const newsDetail = {
+                photo: 'https://mfi.ug.edu.pl' + img,
+                link: 'https://mfi.ug.edu.pl' + href,
+                datetime: datetime,
+                title: title,
+                body: body,
+            };
+            newsArray.push(newsDetail);
+        });
```

Comment:
I think it would be better to change this into something like:
```typescript
const $ = cheerio.load(HTMLData);
        const selectedElem = '.item-list > ul > li';
        const newsArray: News[] = $(selectedElem)
            .map((parentIndex, parentElem) => {
                const img = $(parentElem)
                    .find('.image-style-news-main-page')
                    .attr('src');
                const href = $(parentElem).find('.news-title a').attr('href');
                const title = $(parentElem).find('.news-title a').text();
                const datetime = $(parentElem).find('.news-datetime').text();
                const body = $(parentElem).find('.news-body').text();
                const newsDetail = {
                    photo: 'https://mfi.ug.edu.pl' + img,
                    link: 'https://mfi.ug.edu.pl' + href,
                    datetime: datetime,
                    title: title,
                    body: body,
                };

                return newsDetail;
            })
            .get();
```